src/cache.ts

```diff
@@ -102,6 +108,14 @@ export default class Cache {
         let value;
         if (entry) {
             value = entry.value;
+            if (this.policy.stdTTL > 0) {
+                const deadline = entry.created.getTime() + this.policy.stdTTL * 1000;
+                const now = (new Date()).getTime();
```

Comment:
Instead of (new Date()).getTime() - can you use Date.now() in the if statement below which is functionally the same.  Otherwise, this looks great - thank you for the enhancement!